SuperBuild/External_ITKv5.cmake

```diff
@@ -153,7 +153,7 @@ if(NOT DEFINED ${extProjName}_DIR AND NOT ${USE_SYSTEM_${extProjName}})
   set(${proj}_REPOSITORY ${git_protocol}://github.com/InsightSoftwareConsortium/ITK.git)
   # set(${proj}_REPOSITORY ${git_protocol}://github.com/stnava/ITK.git)
   set(${proj}_GIT_TAG e21a56d1227c5433066237060368cb4532b8a9d2)  # Adaptive denoising module 10/2/2020
-  set(ITK_VERSION_ID ITK-5.1.1) ### NOTE: When updating GIT_TAG, also update ITK_VERSION_ID
+  set(ITK_VERSION_ID ITK-5.1) ### NOTE: When updating GIT_TAG, also update ITK_VERSION_ID
```

Comment:
Circleci is still failing with the same error. I compiled locally to debug and CMake appears to want ITK_VERSION_ID to be ITK-5.2.

ITK-5.1 worked for me but I was building the actual 5.1.1 release tag. Your commit is ahead of that. It's like it's rounding up somehow.

Reply:
Thanks @cookpa .  So what would you recommend?  Should we simply label it as 5.2?

Reply:
That is the immediate fix. I can try to understand it further.

What happens is that CMake writes

`staging/lib/cmake/ITK-5.2/ITKConfig.cmake`

even when the file contains `set(ITK_VERSION_ID ITK-5.1)`.